app/models/open_weather_map.rb

```diff
@@ -0,0 +1,47 @@
+COOLDOWN_IN_MINUTES = 10
+OPEN_WEATHER_MAP_URI = 'http://api.openweathermap.org/'
+OPEN_WEATHER_MAP_PATH = 'data/2.5/forecast/daily'
+OPEN_WEATHER_MAP_CRITERIA = 'zip=22630,us'
+OPEN_WEATHER_MAP_KEY = '3fcb4e23443750cc1256f3016a160ca7'
```

Comment:
we should discuss environment variables today or tomorrow. you never want to put your keys in your code bc people can steal them